params/params_tmpl_am.R

```diff
@@ -0,0 +1,15 @@
+# !! PLEASE DO NOT EDIT THIS FILE !!
+# ** make a copy and edit that instead **
+# ________________________________________________
+# input parameters
+
+# sigma2: 0 < value < 5
+sig2 <- 2.51
+    
+# e.g. "anas_krystallinus"    
+species.name <- " Alekhya's hawk "
```

Comment:
I'm not sure, but the space around the text here might cause problems.

```suggestion
species.name <- "Alekhya's hawk"
```